lib/bridge/MatrixHandler.js

```diff
@@ -250,6 +252,11 @@ MatrixHandler.prototype._onAdminMessage = Promise.coroutine(function*(req, event
                 example: `!removepass [irc.example.net]`,
                 summary: `Remove a previously stored NickServ password`,
             },
+            "!feature": {
+                example: `!feature feature-name [true/false]`,
+                summary: `Enable or disable a feature for your account.` +
+                         `Will display if true/false not given.`,
```

Comment:
display what? (dumb question, I know, but we should say "current value" or whatever)